src/libs/modulegroups.c

```diff
@@ -399,7 +399,9 @@ static void _lib_modulegroups_update_iop_visibility(dt_lib_module_t *self)
         }
         else
         {
-          int is_match = (g_strstr_len(dt_iop_get_localized_name(module->op), -1, text_entered) != NULL);
+          int is_match = (g_strstr_len(g_utf8_casefold(dt_iop_get_localized_name(module->op), -1), -1,
```

Comment:
As we are at it, please makes this a const. Thanks.

Reply:
Done. Thank you.